utils/tests/utils.go

```diff
@@ -768,3 +768,12 @@ func VerifySha256DetailedSummaryFromResult(t *testing.T, result *commandutils.Re
 		}
 	}
 }
+
+func SkipKnownFailingTestIfPossible(t *testing.T) {
+	skipMonth := time.July
```

Comment:
I think it makes more sense to receive the skipping period from the test that is being skipped as it might have a known fixing date.  
And then the name can be more clear - maybe SkipKnownFailingTestUntil()